dashboard/src/components/js/hooks/useOutsideClick/useOutsideClick.test.js

```diff
@@ -30,21 +30,27 @@ describe(useOutsideClick, () => {
   it("should attach the event to the global event listener", () => {
     // Mock addEventListener
     const listeners = {};
-    document.addEventListener = jest.fn((event, cb) => {
-      listeners[event] = cb;
-    });
+    document.addEventListener = jest.fn(
+      (event, cb) => {
+        listeners[event] = cb;
+      },
+      { capture: true },
+    );
 
     mount(<TestComponent />);
-    expect(Object.keys(listeners).length).toBe(1);
+    expect(Object.keys(listeners).length).toBe(2);
```

Comment:
Why did this change?

Reply:
I'm not 100% sure, but it has something to do with the changes in the event delegation [1], but I just manually confirmed the hook was working as expected (that is, clicking outside the context switcher and right menu is working as expected), but I performed this change for the tests to pass.

[1] https://reactjs.org/blog/2020/10/20/react-v17.html#changes-to-event-delegation